Gang Update 5.1

Ikarrus opened the pull request:
- [x] Ready to Merge!

As of Update 5 I declare the game mode "Feature Complete" in that I have reached a point of satisfaction where I no longer feel I need to add any further significant systems or features. Frequency of updates will likely fall as will the amount of changes each will make.
#### Major Updates
- Goal reduced back down to 50%
- Reduced income rate to 3 minute intervals.
- Promotion costs reduced to 10, 30, 50 each.
#### Minor Updates
- Income delay is now a editable var in game for debug and testing purposes
- Income rate no longer scales down if you're over 15 territories
- Increased spraycan use to 20
- Boss icon made more obvious. They're now red Gs
- Better descriptions for distributing spraycans and victory conditions
- Lieutenants receive objectives now
- Poseur tags no longer pick gang B's tag

Ikarrus committed:
Gang Update 5.1

I'm quite satisfied in the current state of gang and I feel that I no longer have any more significant changes to make. Gang 5.1 includes primarily minor tweaks.
- Goal reduced back down to 50%
- Boss icon made more obvious. They're now red Gs
- Better descriptions for distributing spraycans and victory conditions
- Lieutenants receive objectives now
- Increased spraycan use to 20
- Poseur tags no longer pick gang B's tag

Ikarrus committed:
- Tweaks to instructional text
- Promotion costs reduced to 10, 30, 50 influence each
- changelog

WJohn commented:
I feel for consistencys' sake regular gangsters should be red Gs and the bosses should be blue. This is done with wizards and revs, too, though I understand the problem with red/blue gangs being confusing to new players and accidentally have them attacking the boss.

Ikarrus committed:
- Reduced income to 3 minute intervals
- Income delay is now a editable var in game for debug and testing purposes
- Income rate no longer scales down if you're over 15 territories

Ikarrus committed:
Round Integers
